about/index.html

```diff
@@ -0,0 +1,110 @@
+        <a href="/about" class="available">About us</a>
+        <a href="/available">Available Puppies</a>
+        <a href="/reviews">Reviews</a>
+        <a href="/contact">Contact us</a>
+      </nav>
+    </header>
+    <main>
+      <h1>About us</h1>
+<p></p>
+<i class="fa-solid fa-paw"></i>
+<p>
+
+We are not your average breeder! We focus on creating an experience for the families and 
+puppies that will bring joy for years to come.
+<p></p>
+
+How are we different?
+
+<ul>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Gift Bag: We send home a gift bag that includes the popular Snuggle Puppy and many other goodies to help your pup transition to your home</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Communication: Once your adoption application is received, we will provide photos, video, and even Facetime with the puppy as we approach adoption day</li>
+<li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Education: We have a ‘Congrats file’ that’s full of great information, including:</li>
+</ul>
+ <br/>
+</pre>o What’s the best way to travel with my puppy?<br/>
+</pre>o What do I need to know about the drive/flight home?<br/>
+</pre>o We share tips to make the transition to home as smooth as possible, while also creating good long-term habits<br/>
+</pre>o Grooming guidelines<br/>
+</pre>o Where do I start with toys?<br/>
+</pre>o Why crate train?<br/>
+ 
+<ul>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Home Raised: The puppies are always raised in my home, not in a separate kennel or building.</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Socialized: The puppies have many experiences to help them prepare for their new life with you, including other people / pets and activities</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Crate Training: Puppies are crate-trained with their Snuggle Puppy and toys before they come to your 
+    home to ensure a smooth transition. Here are a few reasons why this is such a huge head start:</li>
+
+</ul>
+<ul> 
+ 
+  <br/>
+</pre>o They will eat their food in their crates and have excellent eating habits<br/>
+</pre>o They will eat their food in their crates and have excellent eating habits<br/>
+</pre>o Potty training will be well underway<br/>
+<ul>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Health Tested: Their parents have been health tested with guaranteed pedigree</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Training: We will connect you with an excellent online trainer at a discounted rate. 
+    They have videos and information to walk you through every puppy habit or issue that you can think of.</li>
+    <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Transparency: We encourage our families to connect through our 
+      Facebook group. We want people to be able to ask questions and see littermates if they’d like to! We also confess the best part of our days is when we get to see "pupdates" from our families</li>
+
+</ul>
```

Comment:
These lists look bad:

![image](https://github.com/user-attachments/assets/e46ddf0b-ca0d-48ca-99bd-4fb1a4d2a22c)

A lot of the HTML is invalid. I'm not sure what you're trying to do with `</pre>` etc. Make these into valid `<li>` elements like the other list items, and take out the `o ` you are prepending each item with.